src/shared/utils/challenge-listing/sort.js

```diff
@@ -34,7 +34,18 @@ export default {
     name: '# of submissions',
   },
   [SORTS.PHASE_END_TIME]: {
-    func: (a, b) => a.currentPhaseRemainingTime - b.currentPhaseRemainingTime,
+    func: (a, b) => {
+      function nextSubEndDate(o) {
+        if (("checkpointSubmissionEndDate" in o) && moment(o.checkpointSubmissionEndDate).isAfter()) {
```

Comment:
The first part of condition should be just `o.checkpointSubmissionEndDate && ...`